prime-router/src/main/kotlin/cli/main.kt

```diff
@@ -136,6 +167,7 @@ class RouterCli : CliktCommand(
         }
     }
 
+    // NOTE: This exists to support not-yet-implemented functionality.
```

Comment:
As far as I can tell, `postHttp` is supposed to support `--send`, but it’s not wired up.